Added the starter project

SHIVANGISINGH1 opened the pull request:
### Resolved #9 

I have initialized the starter project using next js.
The commands you need to run after cloning the respository: 

**Make sure you have [node ](https://nodejs.org/en/download/)installed**
<pre>
 
1.   cd community-website
2.   npm install
3.   npm run dev

</pre>

AshAman999 reviewed (approved):
LGTM :)

Rahulkhinchi03 commented:
@SHIVANGISINGH1 please share your command line screenshot for new contributors to understand how to run this. 

Others, LGTM

SHIVANGISINGH1 commented:
> @SHIVANGISINGH1 please share your command line screenshot of running these commands for new contributors to understand how to run this.
> 
> Others, LGTM ☑️

The commands I share above are sufficient for starting the project, after this PR gets merged I will attach the ss of the commands.

SuperAayush commented on community-website/pages/index.js:
```diff
@@ -0,0 +1,69 @@
+import Head from 'next/head'
```
As the file name is index.js can we have a comment above the code in the file specifying which page is this of the website in order to make things organized?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

SHIVANGISINGH1 replied:
Yes, I can perform the clean-up and add files according to our requirements, with proper comments.

SuperAayush reviewed (approved):
Just a suggestion if we could include some comments so that it is easy to understand the code.
In this PR comments in given below two files should be made IMO:

- community-website/pages/index.js
- community-website/styles/Home.module.css


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Unreadable, Efficient

SHIVANGISINGH1 commented:
> Just a suggestion if we could include some comments so that it is easy to understand the code. In this PR comments in given below two files should be made IMO:
> 
> * community-website/pages/index.js
> * community-website/styles/Home.module.css
> 
> __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Tags:** Unreadable, Efficient

I didn't get what you are trying to say. 
Are you suggesting that I put comments to explain the file structure of adding CSS files?

SuperAayush commented:
> > Just a suggestion if we could include some comments so that it is easy to understand the code. In this PR comments in given below two files should be made IMO:
> > 
> > * community-website/pages/index.js
> > * community-website/styles/Home.module.css
> > 
> > __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  
> 
> I didn't get what you are trying to say. Are you suggesting that I put comments to explain the file structure of adding CSS files?

If you can do that, it would be great but mainly I was talking about mentioning which page of the website's CSS file is that so that it is easy to navigate for the folks who will be contributing for the same page.
Does that make sense? If not maybe we can have a chat on Discord Voice Channel.__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Unreadable

SHIVANGISINGH1 commented:
> > > Just a suggestion if we could include some comments so that it is easy to understand the code. In this PR comments in given below two files should be made IMO:
> > > 
> > > * community-website/pages/index.js
> > > * community-website/styles/Home.module.css
> > > 
> > > __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)
> > 
> > 
> > I didn't get what you are trying to say. Are you suggesting that I put comments to explain the file structure of adding CSS files?
> 
> If you can do that, it would be great but mainly I was talking about mentioning which page of the website's CSS file is that so that it is easy to navigate for the folks who will be contributing for the same page. Does that make sense? If not maybe we can have a chat on Discord Voice Channel.__
> 
> Anyways it was just my personal suggestion, I have approved the commits anyhow. __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

Okay @SuperAayush I got what you are saying. Great suggestion btw, it will help others to easily understand the code. 

Will implement the changes asap.

SHIVANGISINGH1 commented:
I have done some clean up, removed the extra files and added comments in the code.
Please review it and let me know if there is anything else to add.

SuperAayush reviewed (approved):
LGTM👍🏻

Rahulkhinchi03 commented:
@Vyom-Yadav Even I am not sure about the changes as I am new to it. Should we just wait for some time or merge it? as other members has already approved it.
